docs/guides/insar_product_guide.md

```diff
@@ -225,13 +225,13 @@ Along with the image files, there are currently two text files - the main readme
 
 There are several options offered with the InSAR products.  Currently, these are (1) the number of looks to take, (2) inclusion of look vectors, (3) inclusion of the line of sight displacement file, and (4) inclusion of the incidence angle map.
 
-The number of looks drives the pixel spacing of the output products. Selecting 10x2 looks will yield larger products with 80 m pixel spacings. Selecting 20x4 looks reduces the resolution to 160 m and reduces the size of the products (roughly 1/4 the size of 10x2 look products). The default is 20x4 looks.
+1. The **number of looks** drives the resolution and pixel spacing of the output products. Selecting 10x2 looks will yield larger products with 80 m resolution and pixel spacing of 40 m. Selecting 20x4 looks reduces the resolution to 160 m and reduces the size of the products (roughly 1/4 the size of 10x2 look products), with a pixel spacing of 80 m. The default is 20x4 looks.
```

Comment:
@talogan Can you verify that these are the correct statements about the resolution and the pixel spacing?